waspc/src/Wasp/Message.hs

```diff
@@ -9,6 +9,6 @@ module Wasp.Message (Message (..), SendMessage) where
 -- This protocol is for sending messages for display in a UI.
 -- If you need success or failure another purpose use return values.
 
-data Message = Start String | Success String | Failure String | Warning String
+data Message = Notification String | Start String | Success String | Failure String | Warning String
```

Comment:
I would rename `Notification` to `Info`, as I feel like that it is used more commonly as a term for this kind of stuff. Most often log levels are info, warn and error.

Reply:
Yeah, I was doubting about that. Note we our protocol already goes beyond info/warning/error. We have "start" already as a special form of notification, and "success"/"failure" in the end. I'll rename it to Info though.